### PR TITLE
People converted to clockcult will actually have their world turn yellow briefly, matching the messages

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Being converted to clockcult will briefly cause your world to turn yellow and you to hear the machines of Reebe, matching the description in the messages.
spellcheck: The messages for being converted and for failing conversion to clockcult have been updated.
/:cl:

Requires #28381
Fixes roundstart antag sounds not working.
north trade post echo 2017-06-12 133649
https://www.dropbox.com/s/goihknghj19aa5q/2017-06-12_14-50-19.mp4?dl=0

### CODE REVIEW

**Comment by GunHog** [comment]
HELP! SOMEONE URINATED ON MY FACE!!

**Comment by lzimann** [comment]
why though

**Comment by RandomMarine** [comment]
DEUS EX
SPACEMAN REVOLUTION

**Comment by RemieRichards** [comment]
@GunHog JARATE

**Comment by Chronitonity** [comment]
That looks eye searingly awful

**Comment by GunHog** [comment]
Suggestion: Replace this with the golden flash that happens with Ratvar and Nar'Sie are fighting! Do it for the same amount of time as the Legion boss' summoning effect!

**Comment by ChangelingRain** [comment]
It IS the golden flash effect, but it lasts a bit. It looks a lot better ingame, since it fades out.

**Comment by GunHog** [comment]
Oh, you really should have made a .gif or something.

**Comment by ChangelingRain** [comment]
https://www.dropbox.com/s/goihknghj19aa5q/2017-06-12_14-50-19.mp4?dl=0

File: code/datums/antagonists/datum_clockcult.dm [hunk 1]
```diff
@@ -50,11 +50,14 @@
...
 			if(R.connected_ai && !is_servant_of_ratvar(R.connected_ai))
-				to_chat(current, "<span class='boldwarning'>You have been desynced from your master AI.</span>")
-			to_chat(current, "<span class='boldwarning'>In addition, your onboard camera is no longer active and you have gained additional equipment, including a limited clockwork slab.</span>")
+				to_chat(R, "<span class='boldwarning'>You have been desynced from your master AI.<br>\
+				In addition, your onboard camera is no longer active and you have gained additional equipment, including a limited clockwork slab.</span>")
+			else
+				to_chat(R, "<span class='boldwarning'>Your onboard camera is no longer active and you have gained additional equipment, including a limited clockwork slab.</span>")
 		if(isAI(current))
-			to_chat(current, "<span class='boldwarning'>You are able to use your cameras to listen in on conversations.</span>")
+			to_chat(current, "<span class='boldwarning'>You are noew able to use your cameras to listen in on conversations, but can no longer speak in anything but Ratvarian.</span>")
```
Comment (Xhuis) [inline code/datums/antagonists/datum_clockcult.dm]: meow* nya

**Comment by DoctoreMad** [comment]
pls no
Saw gif still no

**Comment by XDTM** [comment]
I like it

**Comment by Repukan** [comment]
I created this station of pissy wonder with my magic!

Dare you enter my magical realm?

**Comment by Bawhoppen** [comment]
Looks interesting but a bit bland

**Comment by lordpidey** [comment]
As much as I like this, isn't this in violation of the feature freeze?

**Comment by ChangelingRain** [comment]
It's not a balance change, it's not a new toy/antag type, and it's not civilian gear that can only really be used as an antag.
So no, the freeze doesn't catch this.

File: code/datums/antagonists/datum_clockcult.dm [hunk 2]
```diff
@@ -20,25 +20,28 @@
...
 		. = is_eligible_servant(new_owner.current)
 		if(!silent && new_owner.current)
-			if(issilicon(new_owner.current))
-				to_chat(new_owner.current, "<span class='heavy_brass'>You are unable to compute this truth! Your circuits surge with energy! Your world glows a brilliant yellow!\
-				<br>All at once it comes to you. Ratvar, the Clockwork Justiciar, lies in exile, derelict and forgotten in an unseen realm.</span>")
+			if(.)
+				to_chat(new_owner.current, "<span class='heavy_brass'>The world before you suddenly glows a brilliant yellow. [issilicon(new_owner.current) ? "You cannot compute this truth!" : \
+				"Your mind is racing!"] You hear the whooshing steam and cl[pick("ank", "ink", "unk", "ang")]ing cogs of a billion billion machines, and all at once it comes to you.<br>\
+				Ratvar, the Clockwork Justiciar, [GLOB.ratvar_awakens ? "has been freed from his eternal prison" : "lies in exile, derelict and forgotten in an unseen realm"].</span>")
+				flash_color(new_owner.current, flash_color = list("#BE8700", "#BE8700", "#BE8700", rgb(0,0,0)), flash_time = 50)
```
Comment (Xhuis) [inline code/datums/antagonists/datum_clockcult.dm]: Might wanna update the PR title to include that the messages are different now, too!

**Pull request merged**